Assignment1/styles.css

```diff
@@ -0,0 +1,292 @@
+body {
+  padding: 0px;
+  margin: 0px;
+  font: 100% Poppins, sans-serif;
+}
+
+#upper {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  z-index: 1;
+  color: #1b2021;
+  width: 1000px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+}
+
+nav {
+  color: #fbfef9;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 0px;
+}
+nav #left-nav {
+  font-size: 1.5em;
+}
```

Comment:
as a general rule, add an empty line after each class/id

Reply:
The css file is generated by sass using the scss one. I saw that it applies this rule but when nested elements appear it doesn't add the extra line. Should I add it manually?